### ISSUE TITLE
rxjs-no-unsafe-scope support for static class variable

### ISSUE THREAD

**Issue by kctang** [cartant/rxjs-tslint-rules#68]
I am not sure if these scenarios should be part of `allowProperties` or a new option but it would be useful to be able to use `static` and `static readonly` in observable chains without triggering this rule.

```typescript
class MyClass {
  private static STATIC_VAL = 'bob'
  private static readonly STATIC_RO_VAL = 'bob'

  static_val_notOk () {
    return of(null).pipe(
      concatMap(() => {
        // Unsafe scopes are forbidden
        console.log(MyClass.STATIC_VAL)
        return of(null)
      }),
      concatMap(() => {
        // Unsafe scopes are forbidden
        console.log(MyClass.STATIC_RO_VAL)
        return of(null)
      })
    )
  }
}
```

**Comment by cartant** [comment]
I think the `readonly` failure is a bug and the other should depend upon the `allowProperties` option.
